app/services/rad_nav/admin_menu.rb

```diff
@@ -27,6 +27,7 @@ def standard_items
          sidekiq,
          DropdownMenuItem.new(view_context, 'Company Info', view_context.company_edit_path),
          generate_jwt,
+         calendar,
```

Comment:
make a "Divisions" menu that has 2 dropdown items, Divisions which goes to the index page and Calendar that goes to this page